person.rb

```diff
@@ -7,15 +7,20 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
     @name
   end
 
-  attr_reader :id, :age
+  attr_reader :id, :age, :rentals
   attr_accessor :name
 
+  def add_rental(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] I suggest that this method for adding rentals takes in the `book` and the `date` and the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    
    ```ruby
      def add_rental(book, date)
        Rental.new(date, book, self)
      end
    ```

Reply:
Wow, thanks for the feedback and for taking the time to explain me step by step! I really appreciated that.
Changes are done!